src/tips/models.py

```diff
@@ -100,3 +107,135 @@ def available_tips(cls, user: ClusiveUser, page: str, version_count: int):
                 if h.type.can_show(page, version_count)
                 and h.ready_to_show()]
 
+
+class CallToAction(models.Model):
```

Comment:
We'd discussed in the tech meetings that we might want to include the content for the CTA in the database itself, so that adding new CTAs doesn't require modifying template code. Should this be considered for a future enhancement?

Reply:
Yes, I could imagine in the future that there we could add a "simple" CTA type that would work this way. The current ones all have unique markup, including buttons, images, links, etc, that would not be easy to implement as database content.